### PR TITLE
release notes for Regolith 3.1

### PR DESCRIPTION
**Pull request by kgilmer**
More (minor) changes expected before release.

* Fix regolith-control-center builds on manic
* Finish integrated `childe` for sway

### CODE REVIEW

File: content/docs/reference/Releases/regolith-3.1-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  Release notes for Regolith 3.1
+---
+
+# Regolith 3.0 Release Notes
+
+[Regolith 3.0](https://github.com/orgs/regolith-linux/projects/26) is a minor release focusing on Ubuntu 23.10 support.
+
+## New Major Features
+
+* Ubuntu 23.10 (manic) support
```
Comment (SoumyaRanjanPatnaik) [inline content/docs/reference/Releases/regolith-3.1-release-notes.md:15]: *mantic

**Review by SoumyaRanjanPatnaik — CHANGES_REQUESTED** [review]

File: content/docs/reference/Releases/regolith-3.1-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: "3.1 Release Notes"
+linkTitle: "3.1 Release Notes"
+weight: 2
+description: >
+  Release notes for Regolith 3.1
+---
+
+# Regolith 3.0 Release Notes
```
Comment (SoumyaRanjanPatnaik) [inline content/docs/reference/Releases/regolith-3.1-release-notes.md:9]: 3.1

File: content/docs/reference/Releases/regolith-3.1-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+[Regolith 3.0](https://github.com/orgs/regolith-linux/projects/26) is a minor release focusing on Ubuntu 23.10 support.
```
Comment (SoumyaRanjanPatnaik) [inline content/docs/reference/Releases/regolith-3.1-release-notes.md:11]: 3.1

File: content/docs/reference/Releases/regolith-3.1-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## New Major Features
+
+* Ubuntu 23.10 (manic) support
+
+## Changelog Delta from Regolith 3.0
```
Comment (SoumyaRanjanPatnaik) [inline content/docs/reference/Releases/regolith-3.1-release-notes.md:36]: 3.1

**Review by SoumyaRanjanPatnaik — CHANGES_REQUESTED** [review]

File: content/docs/reference/Releases/regolith-3.1-release-notes.md (hunk 4 above)
Comment (kgilmer) [reply content/docs/reference/Releases/regolith-3.1-release-notes.md:38]: this one actually should be `3.0`

**Comment by kgilmer** [comment]
Ready for another pass.  @SoumyaRanjanPatnaik

**Comment by SoumyaRanjanPatnaik** [comment]
Is this for the beta or the final release?

**Comment by SoumyaRanjanPatnaik** [comment]
I also wonder what the status of the sway session is going to be in this release. It might be a good idea to extensively document the sway session before we release sway as stable.

**Comment by kgilmer** [comment]
> Is this for the beta or the final release?

This is for beta 1.  I had hoped to get this out back in October but the portals thing happened.

> I also wonder what the status of the sway session is going to be in this release. It might be a good idea to extensively document the sway session before we release sway as stable.

What's in this release from 3.0 is in the release notes.  Regarding docs, I'd be happy to help out.  LMK if you have specifics..

**Comment by SoumyaRanjanPatnaik** [comment]
Looks good. I suppose we can proceed with the release :)

**Review by SoumyaRanjanPatnaik — COMMENTED** [review]
I just noticed that `sway-regolith` is missing in this changelog. Also it seems like  `r3_1-beta1` tag (which is built by voulage [here](https://github.com/regolith-linux/voulage/blob/746bc0ef40cb87012a303dda7c9546e6370a021a/stage/testing/package-model.json#L191C1-L194C7) ) points to an older version of `sway-regolith`. However the `r3_1-beta1-ubuntu-mantic` tag does seem to package the correct version [here](https://github.com/regolith-linux/voulage/blob/746bc0ef40cb87012a303dda7c9546e6370a021a/stage/testing/ubuntu/mantic/package-model.json#L27C1-L30C7)

**Comment by kgilmer** [comment]
Updated tags on `sway-regolith` and added changelog

**Pull request merged**